### ISSUE TITLE
Feature Request: Add ALT text for images

### ISSUE THREAD

**Issue by bekopharm** [Automattic/wordpress-activitypub#704]
### What

Apparently the ALT text for images is not added to ActivityPub - I'm almost sure this worked before so uncertain if this is a bug or a feature request.

### Why

This will solve my accessibility needs :upside_down_face: 

### How

I recently posted this:

> https://beko.famkos.net/2024/03/04/15629077/

The theme correctly renders `alt="An open Thermomix with freshly grinded beans next to a bag of coffee beans."`

The syndicated Mastodon note appears to be missing the ALT text:

> https://social.tchncs.de/@beko@beko.famkos.net/112037681223977248

The image was added with the Block Editor:

```
<!-- wp:image {"id":15629076,"sizeSlug":"large","linkDestination":"media"} -->
<figure class="wp-block-image size-large"><a href="https://beko.famkos.net/wp-content/uploads/2024/03/1709557940705.jpg"><img src="https://beko.famkos.net/wp-content/uploads/2024/03/1709557940705-1024x576.jpg" alt="An open Thermomix with freshly grinded beans next to a bag of coffee beans." class="u-photo wp-image-15629076"/></a></figure>
<!-- /wp:image -->
```

Activity settings are as following:
![image](https://github.com/Automattic/wordpress-activitypub/assets/5016428/c9a099af-eb69-4564-b2e9-2992fdad7c7d)

**Comment by pfefferle** [comment]
It worked before we added the Block-Support and will work again with the next version: https://github.com/Automattic/wordpress-activitypub/pull/661/files#diff-51c484678979200602098df73756690a869c7322024f17c7113b368acff20139R282

:)

**Comment by bekopharm** [comment]
Why do I even bother making tickets when you solve em before the fact :zany_face:
